tests/native_tests/language.hy

```diff
@@ -1420,31 +1420,48 @@ cee\"} dee" "ey bee\ncee dee"))
   (assert (= y 1)))
 
 (defn test-require []
-  "NATIVE: test requiring macros from python code"
```

Comment:
It doesn't look like this functionality is tested elsewhere; if we're still planning on supporting macros defined in python we should still have a test for this?

Reply:
I don't think it was ever something we wanted to support per se. The only motivation I can think of for writing those definitions in Python was to try to make the test independent of the Hy syntax for defining macros. Maybe that made more sense earlier in Hy's life, but now, it's the implementation we expect to change more than the Hy syntax.